sysdump/sysdump.go

```diff
@@ -190,7 +190,7 @@ func NewCollector(k KubernetesClient, o Options, startTime time.Time, cliVersion
 	if c.Options.CiliumNamespace == "" {
 		ns, err := detectCiliumNamespace(k)
 		if err == nil {
-			c.log("🔮 Detected Cilium installation in namespace %q", ns)
+			c.log("🔮 Detected Cilium installation in namespace: %q", ns)
```

Comment:
**note:** I made some changes here to make the log messages consistent/conventional. I limited changes to a small radius (same screen, similar context), but could have gone wild with changes...